dasharo-compatibility/dcu.robot

```diff
@@ -158,5 +156,46 @@ Prepare DCU Test Environment
     ...    6e5a6722955e4f78d947654630f27ff833703fbc04776ffed963c96617f6bb2a
 
     Run    cp ${FW_FILE} dcu/coreboot.rom
+    Run    chmod -R a+rw dcu
     ${local_path}=    Join Path    ${DL_CACHE_DIR}    logo.bmp
     Run    cp ${local_path} dcu/logo.bmp
+
+Write SMMSTORE State 1
```

Comment:
The keyword name should be self-descriptive. This name does not tell us what it does.

Reply:
I have removed this keyword entirely. It's purpose was to be able to easily modify the set of options that we will check if are modified properly but it's not needed now as modifying only one option is enough. 
https://github.com/Dasharo/open-source-firmware-validation/pull/543/commits/74692a0053682dfeb92db24d60391ce352f1d17b#diff-7ee5e21e59cd627383812396fcf45a6e7ecb3e2f096ab5ae5203a0c6bd75989fL109